.github/workflows/create-pr.yaml

```diff
@@ -19,6 +20,51 @@ permissions:
   contents: read
 
 jobs:
+  create-tentants-config-pr:
+    if: >
+      inputs.run_for == 'tentants-config' || inputs.run_for == 'both' || inputs.run_for == ''
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout tenants-config
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        repository: redhat-appstudio/tenants-config
+        ref: main
+        path: tenants-config
+
+    - name: Update ec-policies
+      run: ./hack/update-infra-deployments.sh ../tenants-config
+      working-directory: ec-policies
+
+    - name: Checkout infra-deployments-ci
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        path: infra-deployments-ci
+
+    - name: Create PR in infra-deployments
+      env:
+        EC_AUTOMATION_KEY: ${{ secrets.EC_AUTOMATION_KEY }}
+        DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
+        APP_INSTALL_ID: 32872589
```

Comment:
Remember that we need to request the install of the ec automation app to tenants-config repository as well ([here](https://github.com/apps/ec-automation/installations/32872589)), make sure we don't loose the access to infra-deployments and build-definitions in the process...

Reply:
I added a deploy key for tenants config under the org. The tenants-config fork should now have the proper keys configured.